### ISSUE TITLE
WeakAuras listing updates ready, but never applied.

### ISSUE THREAD

**Issue by JVemmer** [WeakAuras/WeakAuras-Companion#1705]
**Operating System:**
- [x] Windows
- [ ] Mac
- [ ] Linux

**Companion Version:**
4.1.3

**Companion App Console Log:**
```
buildAccountList
LandingPage.vue:1664 validateWowpath
LandingPage.vue:1711 buildVersionList
LandingPage.vue:1800 buildAccountList
utilities.ts:53 Plater not found at D:\Games\World of Warcraft\_retail_\Interface\addons
LandingPage.vue:480 updaterHandler: update-not-available
LandingPage.vue:480 updaterHandler: checkForUpdates
LandingPage.vue:516 Creating default directory
LandingPage.vue:1237 promisesWagoCallsComplete
LandingPage.vue:1266 promisesWagoDataCallsComplete
LandingPage.vue:1307 no data received for luckyoneWarrior
LandingPage.vue:1350 writeAddonData
LandingPage.vue:1237 promisesWagoCallsComplete
LandingPage.vue:1266 promisesWagoDataCallsComplete
LandingPage.vue:1307 no data received for luckyoneWarrior
LandingPage.vue:1350 writeAddonData
```

**WeakAuras Version:**
5.3.4

**WeakAuras Errors:**
No errors related to WeakAuras, just a single one for Wowhead Looter.

**Describe the bug**
Similar bug to #1689. WeakAuras and the Companion keeps telling me to update [LuckyoneUI - Druid](https://wago.io/LuckyoneUI-Druid) and [Dragonriding UI - Luckyone Edit](https://wago.io/LuckyoneUI-Dragonriding), but they are both updated. Uninstalling them, trying to update again, reimporting, nothing works to remove them from the "Ready to Update" list in-game and "Update Ready" status in the companion app.
I have also tried removing the auras, clicking the "Update aura" button in-game, then manually update each aura (right-click and use the dropdown), which gives me message in my chatlog: "WeakAuras:  Invalid update target, uids don't match.". I have also tried to reset my WeakAuras companion and settings.

**To Reproduce**
1. Import one of those WeakAuras
2. Observe behavior

_Note: It is very difficult for me to reproduce it, since I cannot actually make it go away to test the reproduction steps; these steps are merely what I did._

**Expected behavior**
Aura is updated when button is pressed, and update disappears when aura is removed completely.

**Screenshots**
![image](https://user-images.githubusercontent.com/8568258/211090706-33714402-72d2-4604-a431-df3f71d801af.png)

**Include a .zip file containing:**
[weakauras_error.zip](https://github.com/WeakAuras/WeakAuras-Companion/files/10362773/weakauras_error.zip)

**Comment by Likeyn** [comment]
I'm having the same issue with a few WeakAuras and I've noticed something, so I took a few screenshots which I hope may help.

The companion shows the updates and the versions seem to match
![wa-companion-updates](https://user-images.githubusercontent.com/815044/211154439-abbe57d5-7654-43c2-8b50-44d9936c7012.png)

On Wago.io it shows the following patch note for v3.0.19-43 of the WA:
![wa-version-patchnote](https://user-images.githubusercontent.com/815044/211154460-059edfee-ca6c-4d9a-ab23-0a66b41f8915.png)

In-game however, the correct patch note is shown on the WA to update _but_ version and WA URL are those of the previous version (the one which is currently installed in my case):
![wa-addon-updates](https://user-images.githubusercontent.com/815044/211154456-0f104b87-f51e-4bba-9b9b-691850cad2c8.png)

Could it be that the in-game WA addon tries to update to the previous version instead of the last one, or actually updates it but keeps the old "version label" somehow?

**Comment by JVemmer** [comment]
This may have been a case of user error. I found an aura, that I must have duplicated from an old version of those auras, in order to change it, that still had an old link. Once I removed the link there, restarted game and WeakAuras companion, the updates were no longer listed.
There was nothing in the game or WeakAuras companion that helped me find this, I noticed it because I wanted to update the link for upload.

**Comment by rbtmarshall** [comment]
I had this same issue.

When I first found out about WeakAuroras, I went to the wago.io site and downloaded the Wago app thinking that was the only way to get WeakAuroras.  So I installed it and also installed some WeakAuroras.

I was already using the CurseForge to manage my addons in WoW and noticed a conflict with differing the versions between curseforge and wagoapp.   I noticed on wago.io thatthere was also a WA Companion app that only worked with the WeakAuroras and not all the other WoW addons.  So I uninstalled the WagoCompanion app and installed WAcompanion instead keeping curse forge as my defaul WoW addon manager for now.

I kept running into the same issue with the WA ingame app showing the older version needed updated, even after I would update it.


I'm not sure where eak Aurora downloads are stored locally, so I took an alternate route to attempt what I was thinking would be the issue.  two versions of the WA stuck in the addons somewhere.

I finally got the issue resolved by reinstalling the Wago App, uninstalling the older version of  the app that was stuck in "Ready to Update" and relaunched WoW, and that ready-to-update version is now gone .


====
TLDR version;
If you are currently using the WA Companion app
reinstall the Wago Companion app
delete the version of the WA that is stuck in Ready To Update
restart WoW and the version stuck in ready to update should be gone.
